test/Microsoft.AspNetCore.Authentication.Test/CookieTests.cs

```diff
@@ -26,6 +26,416 @@ public class CookieTests
     {
         private TestClock _clock = new TestClock();
 
+        [Fact]
+        public async Task CanForwardDefault()
+        {
+            var services = new ServiceCollection().AddLogging();
+
+            services.AddAuthentication(o =>
+            {
+                o.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                o.AddScheme<TestHandler>("auth1", "auth1");
```

Comment:
Why is this call nested inside of AddAuthenticaton instead of being:

```C#
services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
        .AddScheme<TestHandler>("auth1", "auth1")
        .AddCookie(o => o.ForwardDefault = "auth1");
```

Reply:
It could go in either place for the purposes of these tests, but the original intent was to exercising that you can forward to non-security based IAuthenticationHandlers

Reply:
The difference between the two places is that adding schemes directly to the option is coming from Core Authentication (httpabstractions), while the AuthenticationBuilder is something that is part of Security and adds a bunch of constraints like deriving from `AuthenticationSchemeOptions/AuthenticationHandler`